index.js

```diff
@@ -1,124 +1,8 @@
 'use strict';
 
-var GetIntrinsic = require('get-intrinsic');
```

Comment:
This file should ideally be compared with light.js since it was moved and then recreated but github is not smart enough, I tried with multiple commits but with no success